bin/reth/tests/commands/bitfinity_node_it.rs

```diff
@@ -311,27 +377,23 @@ pub mod eth_server {
 
         #[method(name = "getLastCertifiedBlock", aliases = ["ic_getLastCertifiedBlock"])]
         async fn get_last_certified_block(&self) -> RpcResult<CertifiedResult<Block<H256>>>;
-
     }
 
+    pub static TXS_ORDER: Mutex<Vec<B256>> = Mutex::const_new(Vec::new());
+
     #[derive(Debug)]
     pub struct EthImpl {
         pub gas_price: u128,
         pub max_priority_fee_per_gas: u128,
     }
 
     impl EthImpl {
-        pub fn new() -> Self {
+        pub async fn new() -> Self {
+            TXS_ORDER.lock().await.clear();
```

Comment:
clearing it does not solve concurrency issues. you should definitely remove the static TXS_ORDER var and substitute it with a new field in the EthImpl struct

Reply:
You right. Replaces with field in EthImpl